code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -687,10 +687,29 @@
 
 	if(istype(user,/mob/living/carbon/human))
 		var/mob/living/carbon/human/H = user
-		if(H.species.can_shred(H))
-			attack_generic(H, rand(30,50), "slashed")
-			return
-
+		//if(H.species.can_shred(H))
+		//	attack_generic(H, rand(30,50), "slashed")
+		//	return

```

Comment:
These lines will need to be marked as //VOREStation Removal, so that they are not uncommented accidentally should Polaris edit these lines.

Reply:
I was gonna comment that when this PR went up but I was uncertain.

Reply:
Thanks, new to this, so I've added the comment to indicate the removal.